tasks/json/parser.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * You need to write a parser, which takes the ./test.json file
+ * and will create a new JSON file with the name parsed.json with the structure:
+ * [
+ *  {
+ *    "docId": "http://doc.epam.com/077b1523-56f0-492a-b954-1269e3acc191"
+ *  },
+ *  {
+ *    "docId": "http://doc.epam.com/077d3a4c-a2fe-4ab9-9046-89c0ce5c5b38"
+ *  }
+ *  .....
+ * ]
+ * where 077b1523-56f0-492a-b954-1269e3acc191 - is the value of the name attribute from the ./test.json file
+ * Please NOTE, that you should omit the .html extension
+ * Please do the task in the scope of the file. Create a pull request a share it with your mentor.
+ */
+const myObj = require ('./test.json');
+const fs = require('fs');
+let allEntries = myObj.list.entries;
+
+let allEntries1 = [];
+ allEntries.forEach(obj => {
```

Comment:
Your solution works, but it is not the optimal one, moreover you do lots of unnecessary magic here :)
1.) Try to use map() function:
const obj = myObj.list.entries.map(item => {
let name = item.name // here is needed name attribute
let neededName = ... replace and concat and othere stuff;
...
...
return {docId: neededName }
})

Reply:
if I user let name = item.name , type of variable 'name' is undefined
Could you please clarify what do you mean under '// here is needed name attribute' and how to use it?

Reply:
Sorry,
let name = item.entry.name -> I mean that it is the way to take the needed id from the json object. And after it, you can work with it.

Reply:
done